README.md

```diff
@@ -17,9 +17,9 @@ To run Kitesocial, you will need:
 
 - Ruby 2.7.6
 - Rails 6.0.3
+- Node 16.12.0
 - Yarn 1.19.1
 - Chrome or Chromium
-- Python 2.7 (to install node-gyp)
```

Comment:
Have you managed to get around this req?

Reply:
🤦 I should read the readme...

Reply:
@dabarrell So the way I confirmed this was to remove Python 2 entirely from my system, rm -rf node_modules, and then run `bin/setup` again, and it seemed to be good!